tasks/app/controllers/tasks_controller.rb

```diff
@@ -5,7 +6,38 @@ def index
 
   def show
     id = params[:id]
-    @num_tasks = 11
+    @next_id = find_next_id(id.to_i)
     @task = Task.find(id)
   end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    Task.create(task_params[:task])
+    redirect_to "/"
+  end
+
+  def destroy
+    id = params[:id]
+    Task.delete(id)
+    redirect_to "/"
+  end
+
+# Finds the id of the next item in the task list
+  def find_next_id(current_id)
```

Comment:
This could also be in the `private` section since you are only using it within this controller